add testing of apache config (#878)
owncloud/docs-server: modules/admin_manual/pages/installation/quick_guides/ubuntu_22_04.adoc

EOM
----

> ==== Test the configuration

Reviewer: ```suggestion
==== Test the Configuration
```

[source,bash]
----
apachectl -t
----

> At this point you would expect the following output

Reviewer: ```suggestion
At this point, the following output is expected:
```
